spec/controllers/presentations_controller_spec.rb

```diff
@@ -55,29 +55,12 @@
 
     before { mock_pundit_user_as(non_admin) }
 
-    describe 'GET #new' do
```

Comment:
It may be a good idea to continue to test these controller endpoints.
Controller specs give you access to a response you can assert against.
e.g. `expect(response.status).to eq(401)` or `expect { subject }.to redirect_to(unauthorized_path)`